src/microbe_stage/organelle_components/CiliaComponent.cs

```diff
@@ -96,6 +114,30 @@ protected override void CustomAttach()
         }
 
         SetSpeedFactor(Constants.CILIA_DEFAULT_ANIMATION_SPEED);
+
+        var microbe = organelle.ParentMicrobe!;
+
+        attractorArea = new Area
+        {
+            SpaceOverride = Area.SpaceOverrideEnum.Combine,
+            GravityPoint = true,
+            GravityDistanceScale = Constants.CILIA_PULLING_FORCE_FALLOFF_FACTOR,
+            Gravity = Constants.CILIA_PULLING_FORCE,
+            CollisionLayer = 0,
+            CollisionMask = microbe.CollisionMask,
+        };
+
+        var sphereShape = new SphereShape { Radius = Constants.CILIA_PULLING_FORCE_FIELD_RADIUS };
+        attractorAreaShape = new CollisionShape { Shape = sphereShape };
+        attractorArea.AddChild(attractorAreaShape);
+        organelle.OrganelleGraphics.AddChild(attractorArea);
```

Comment:
Is it safe in Godot to add an Area to a scaled Node? Adding sub collision shapes or rigid bodies will break stuff. So I worry that this might be a problem as well. To workaround that this area would need to be parented to the base microbe (with potentially an offset if the area is wanted to match where the cilia is actually at).

Reply:
This has a nice effect that the Area's radius scales as the organelle grows, perhaps we can keep this behavior even after moving the Area out of OrganelleGraphics?

Reply:
It would require manual updating code for it, but I'm not against having that as a feature. Though the effect is very temporary as organelles grow pretty fast, that it isn't super important (and with the eventual change to organelles no longer duplicating over time, this feature will just disappear).

Reply:
I've kept this behavior in the latest commit, it's not that hard to add anyways.